django/contrib/admin/static/admin/js/popup_response.js

```diff
@@ -3,6 +3,9 @@
     'use strict';
     var initData = JSON.parse(document.getElementById('django-admin-popup-response-constants').dataset.popupResponse);
     switch(initData.action) {
+    case 'add':
```

Comment:
Without this js change an empty entry would get added to the foreign key list because the `default` case is currently set to `dismissAddRelatedObjectPopup` function as depicted in the pictures below. <br />
I think we should change this default behavior anyways, if not in this ticket. The `add` case is not something that should happen by default IMO.<br />
![screenshot 16](https://cloud.githubusercontent.com/assets/4801628/13298861/87b178aa-db5e-11e5-9995-a881f4afc018.png)
![screenshot 17](https://cloud.githubusercontent.com/assets/4801628/13298862/87b317a0-db5e-11e5-84ec-507da236970c.png)